src/conduit/article/tests/test_article_model.py

```diff
@@ -0,0 +1,40 @@
+"""Tests for the Article model."""
+
+from conduit.article.models import Article
+from conduit.auth.models import User
+from conduit.openapi import json_renderer
+from conduit.scripts.populate import ARTICLE_FOO_ID
+from pyramid.testing import DummyRequest
+from sqlalchemy.orm.session import Session
+
+import json
+
+
+def test_by_shortcuts(db: Session, democontent: None) -> None:
```

Comment:
I'm not sure if `democontent: None` is the right way. I'd do something like `democontent: Any = None`

Reply:
Well it's not. The `democontent` will always be `None` because the `def democontent()` fixture does not return anything. And it's always better to be more specific.

Reply:
Alright.